addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19518,6 +19523,12 @@ msgctxt "#36179"
 msgid "When a file is scanned into the library it will display the metadata title instead of the file name."
 msgstr ""
 
+#. Description of setting with label #29995 "Allow movies in 'Disc n' subdirectories"
```

Comment:
Now that we support Disc, Disk and localized version according to Kodi language settings, you need to extend this I guess.